spec/spec_helper.rb

```diff
@@ -11,8 +11,8 @@
 require 'grocer'
 
 RSpec.configure do |config|
+  # config.filter_run :focus
   config.run_all_when_everything_filtered = true
-  config.filter_run :focus
```

Comment:
I'm not sure why this change is needed. [RSpec will look for `:focus`](https://www.relishapp.com/rspec/rspec-core/docs/filtering/inclusion-filters) metadata to run only a specific spec/group. If no `:focus` is found, everything is run by default.